index.html

```diff
@@ -9,7 +9,7 @@
       <!--by bringing it to its boiling point-->
     </p>
   </section>
-  <section id="manifesto">
+  <section id="manifesto"
```

Comment:
@thhck careful here removing this. probably didn't mean to remove it

```suggestion
  <section id="manifesto">
```

Reply:
thanks good catch